### PR TITLE
Add sidebar left and replace primer with tailwind

### PR DESCRIPTION
**Pull request by samuveth**
- [x] finish replacing all primer classes
- [x] fix markdown styling
- [x] replace primer tooltip with our own
- [x] replace primer anim-pulse and anim-scale-in
- [x] remove primer & support packages

### CODE REVIEW

**Comment by samuveth** [comment]
[Add sidebar left and tailwindcss](https://trello.com/c/s6COWWvF/292-add-sidebar-left-and-tailwindcss)

**Comment by samuveth** [comment]
[Add sidebar left and tailwindcss](https://trello.com/c/s6COWWvF/292-add-sidebar-left-and-tailwindcss)

**Review by bonustrack — CHANGES_REQUESTED** [review]
Wow great work, you're almost done! I found some issues but small ones, mostly about spacing / alignment here is the list:
- On the plugins page the name of the plugins are not aligned
image
- The icon for login is not properly aligned
image
- Sidebar should be hidden on small screen
- There should be some space between title and preview of a proposal here:
image
- The strategies icons not aligned here: 
image
- The bg color of the loading placeholder should use the skin var, now it's the same everywhere sometime it doesn't render well see:
image
- Sidebar bg should use this bg-skin-block-bg color:
image

File: package.json [hunk 1]
```diff
@@ -63,12 +63,15 @@
     "@vue/compiler-sfc": "^3.2.1",
     "@vue/eslint-config-prettier": "^6.0.0",
     "@vue/eslint-config-typescript": "^7.0.0",
+    "autoprefixer": "^10.3.3",
```
[the remark this answers is not in the document]
Comment (samuveth) [reply package.json:66]: Yeah, I'm keeping them until the end incase I need them for testing.

**Comment by samuveth** [comment]
Cool, fixed those

> Sidebar bg should use this bg-skin-block-bg color:

It does use that already, no?

**Comment by bonustrack** [comment]
> It does use that already, no?
Nope it doesn't, i mean about the main sidebar. It should be white here:
image

**Comment by bonustrack** [comment]
Others things i forgot:
- Not a fan of this:
image
I'd rather keep it the way is it was before, with full rounded pill and without "Open".

- I think for now we should show the space that you follow on the main page too. Main page give extra information (numbers of active proposals). We can remove it later when we change some others behavior.

**Comment by bonustrack** [comment]
- Background color shouldnt change on mouse over:
image

**Comment by metrox-eth** [comment]
Capture d’écran 2021-09-05 à 15 06 56
Howdy ! Noticied that the incremential selector is overlapping the pill on mouse over.

**Comment by metrox-eth** [comment]
Capture d’écran 2021-09-05 à 15 30 07
Slightly larger line spacing here

Capture d’écran 2021-09-05 à 15 22 53
Log out text color style is missing

**Comment by samuveth** [comment]
> Sidebar bg should use this bg-skin-block-bg color:

You mean `bg-skin-block-bg` or `bg-skin-header-bg` because where you are pointing the arrow is `bg-skin-header-bg`

**Comment by samuveth** [comment]
> * I'd rather keep it the way is it was before, with full rounded pill and without "Open".

It doesn't look very good without the border around the space image.

**Comment by samuveth** [comment]
> * Background color shouldnt change on mouse over:

I added that because currently it's looks very flat, with the join button being very prominent and the user could miss that you can also view a space by clicking on it.

**Comment by metrox-eth** [comment]
Capture d’écran 2021-09-05 à 15 45 13

Slightly different line spacing add over the height of this section.

Capture d’écran 2021-09-05 à 15 42 45

Missing element here

**Comment by samuveth** [comment]
@metrox-eth thanks a lot for your help!

**Comment by bonustrack** [comment]
@samuveth i mean skin block bg

**Comment by metrox-eth** [comment]
Capture d’écran 2021-09-05 à 18 18 08

Spacing btw address and pill


Capture d’écran 2021-09-05 à 18 23 44

Upload text background missing


Capture d’écran 2021-09-05 à 18 26 16

Placeholder text color


Capture d’écran 2021-09-05 à 18 27 39

Placeholder text color and title bg height

**Comment by metrox-eth** [comment]
Capture d’écran 2021-09-05 à 18 37 05

Title size

**Comment by bonustrack** [comment]
- SafeSnap layout is broken test here: https://bafybeigfqj3vc6qpq5gustfudjtal6r4p2dsx6ksc2czw5h3th6cjngmau.on.fleek.co/#/fabien.eth/proposal/QmV2TVKZmgXHuzT36ockuJ3hwCY31weotgirYTsA18SBDB In proposal creation too test here: https://bafybeigfqj3vc6qpq5gustfudjtal6r4p2dsx6ksc2czw5h3th6cjngmau.on.fleek.co/#/fabien.eth/create
- Add background color from the skin on sidebar space icon so i doesnt look like this:
image
- Try reduce slightly the space between space icon in sidebar
- Check if something is wrong with 3box? Can't see my image here: https://bafybeigfqj3vc6qpq5gustfudjtal6r4p2dsx6ksc2czw5h3th6cjngmau.on.fleek.co/#/fabien.eth/proposal/QmUQ3ytu3JQdbVScFVkEsPtPHc6REEKCtYb7wTcaKkukrR

**Comment by bonustrack** [comment]
Also something wrong with progress bar on block results. We should use the primary color then reduce by 15% opacity for each strategy.

**Comment by samuveth** [comment]
@bonustrack ready for merge from my side. I will improve the progress bar and tooltip in another PR.

**Comment by bonustrack** [comment]
Border shouldn't appear on mobile here:
image

**Comment by bonustrack** [comment]
Same for proposal creation, we shouldnt have this right border on mobile
image

**Comment by bonustrack** [comment]
Also can you reduce the space in green and increase in red? It should look same as on snapshot.org
image

**Comment by bonustrack** [comment]
Feel free to merge once you've done these changes

**Comment by samuveth** [comment]
> Feel free to merge once you've done these changes

@bonustrack Ok please approve

**Review by bonustrack — APPROVED** [review]

**Pull request merged**